text/0118-search-api.md

```diff
@@ -460,14 +460,182 @@ Search queries using `_geoPoint` returns a `_geoDistance` field containing the d
 - Type: Object
 - Required: False
 
-`_formatted` returns highlighted and cropped attributes specified in `attributesToHighlight` and/or `attributesToCrop` of a search result.
+`_formatted` is an object returned in the search response, only if at least one of the following paramaters has been set in the search query:
+- `attributesToHighlight`
+- `attributesToCrop`
+
+If `attributesToHighlight` and `attributesToCrop` are not set, `_formatted` is not returned.
+
+This `_formatted` object will be present in each returned document in the `hits` field.
+
+Example:
+
+```json
+{
+    "q": "",
+    "attributesToCrop": ["title"]
+}
+```
+
+```json
+{
+    "hits": [
+        {
+            "id": 2,
+            "title": "Pride and Prejudice",
+            "_formatted": {
+                "id": "2",
+                "title": "Pride and Prejudice"
+            }
+        },
+        {
+            "id": 456,
+            "title": "Le Petit Prince",
+            "_formatted": {
+                "id": "456",
+                "title": "Le Petit Prince",
+            }
+        }
+    ],
+    ...
+}
+```
+
+Which attributes are present in `_formatted`?
+
+The `_formatted` object will contain attributes coming from the original document, depending on the parameters the users set during the search query. Indeed, **the attributes present in `_formatted` are the addition of the attributes present in `attributesToRetrieve`, `attributesToHighlight`, and `attributesToCrop`**.
+
+Kmowing the default value of `attributesToRetrieve` is `["*"]` (so all the attributes present in `displayedAttributes`), if no `attributesToRetrieve` are set in the search query, `_formatted` will return all the `displayedAttributes`.
+
+Returning attributes in the `_formatted` object does not mean these attributes will be necessarily highlighted or cropped, see the next point.
 
-- If `attributesToHighlight` and `attributesToCrop` are not set, `_formatted` is not returned.
+Which attributes are highlighted or cropped in `_formatted`?
+
+No matter how many attributes are retrieved in `_formatted` following the previous rule:
```

Comment:
What did you mean here?

Reply:
Previously we defined which attributes are defined in `_formatted`. Here, no matter which attributes are present in `_formatted`, the following rules are applied.

Reply:
I did an updated, tell me if it's better